lib/src/main/java/uk/gov/di/ipv/core/library/domain/UserIssuedDebugCredential.java

```diff
@@ -12,10 +12,10 @@
 public class UserIssuedDebugCredential {
 
     DebugCredentialAttributes attributes;
-    private Map<String, Object> gpg45Score;
+    private Map<String, Object> evidence;
```

Comment:
Happy to merge this for now to get things working, but this might be something we should look into for the best name for this. I'm not 100% sure but I think "evidence" is for  only specifically evidence cri types. There are others so having all of the cri gpg45scores under a field called "evidence" might not make sense.